.github/workflows/node.js.yml

```diff
@@ -5,9 +5,12 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ master ]
+    branches:
+      - master
   pull_request:
-    branches: [ master ]
+    branches:
+      - master
+      - fastify-v4
```

Comment:
can this be removed or is still useful?

Reply:
It is useful to run tests, I will remove it before merge the fasting-v4 branch in master